ClientApp/package-lock.json

```diff
@@ -1,8 +1,17650 @@
 {
   "name": "asp-net-core-vue-starter",
```

Comment:
Revert changes for this file: ClientApp/package-lock.json

Reply:
Done